backend/src/app.ts

```diff
@@ -86,6 +86,7 @@ app.use(passport.authenticate('session'));
 
 // Passport config
 passport.use(User.createStrategy());
+// @ts-expect-error  Error from mismatch between User model and Express.User types. Implemented same way as passport-local-mongoose docs suggest
```

Comment:
What type of error did you get and how can I reproduce it? If this is a TODO, will you reference it in an Issue please 🙂

Reply:
If you remove that comment you should see the error on the next line again. It's like the linter ignore comments. I'll copy the error below for reference. 
```
$ npm run build

> backend@1.0.0 build
> tsc

src/app.ts:90:24 - error TS2769: No overload matches this call.
  Overload 1 of 4, '(fn: (user: User, done: (err: any, id?: any) => void) => void): void', gave the following error.
    Argument of type '(user: UserType, cb: (err: any, id?: any) => void) => void' is not assignable to parameter of type '(user: User, done: (err: any, id?: any) => void) => void'.
      Types of parameters 'user' and 'user' are incompatible.
        Type 'User' is missing the following properties from type 'UserType': fname, lname, email, comparePassword, and 69 more.
  Overload 2 of 4, '(fn: (req: IncomingMessage, user: User, done: (err: any, id?: unknown) => void) => void): void', gave the following error.
    Argument of type '(user: UserType, cb: (err: any, id?: any) => void) => void' is not assignable to parameter of type '(req: IncomingMessage, user: User, done: (err: any, id?: unknown) => void) => void'.
      Types of parameters 'user' and 'req' are incompatible.
        Type 'IncomingMessage' is missing the following properties from type 'UserType': fname, lname, email, id, and 70 more.

90 passport.serializeUser(User.serializeUser());
                          ~~~~~~~~~~~~~~~~~~~~



Found 1 error in src/app.ts:90
```

Reply:
This isn't a TODO for me. The way you did it looks copied straight from the docs and it works. The problem here seems to be some janky mismatch between @types/passport and @types/passport-local-mongoose, and I don't think it's worth the time to fix.

Reply:
Ah I see. I had a similar issue on my access middleware PR but figured out a solution for it. But yeah if I did it the way the docs says, let's just keep it as is.